canvas_modules/common-canvas/src/common-properties/panels/tearsheet/tearsheet.scss

```diff
@@ -0,0 +1,21 @@
+.properties-tearsheet-panel {
+	.bx--modal-header {
+		position: absolute;
+		right: 0;
+	}
+	.bx--modal-content {
+		padding: 0;
+		margin: 0;
+		.properties-tearsheet-header {
+			padding: $spacing-06 $spacing-07;
+			border-bottom: 1px solid $gray-20;
+		}
+		.properties-tearsheet-body {
+			padding: $spacing-06 $spacing-07;
+			background-color: $gray-10;
+			.react-codemirror2, .react-codemirror2 .CodeMirror {
+				background-color: $white-0;
```

Comment:
hmm, I don't think this is only for the expression control.  What if other controls are in the tearsheet?